packages/blip/app/components/patient-data.less

```
@@ -453,9 +453,11 @@ a.js-home {
   }
 
   .dashboard-divider {
-    border-left: 1px solid grey;
+    min-width: 0px;
+    padding: 0px;
```

when value is 0, units are not necessary
```suggestion
    min-width: 0;
    padding: 0;
```